libraries/AP_HAL_ChibiOS/hwdef/f103-periph/hwdef.dat

```diff
@@ -114,7 +114,7 @@ define HAL_NO_GCS
 define HAL_NO_LOGGING
 define HAL_NO_MONITOR_THREAD
 
-define HAL_MINIMIZE_FEATURES 0
+define HAL_MINIMIZE_FEATURES 1
```

Comment:
this was deliberately set to not minimize features. It has 50k free flash in master. Why does this PR make it overflow?

Reply:
I have no idea - I was hoping you could tell me! 😄

Reply:
Removed